### ISSUE TITLE
Current page not detected properly by menu

### ISSUE THREAD

**Issue by stancl** [tightenco/jigsaw-docs-template#33]
![Screenshot from 2019-09-18 09-04-27](https://user-images.githubusercontent.com/33033094/65123658-058ec100-d9f4-11e9-8b78-a6dc574c761e.png)

On *Middleware Configuration* site, the *Configuration* item is also active.

**Comment by bakerkretzmar** [comment]
@stancl just to confirm, what are the complete URL paths of your 'Configuration' and 'Middleware Configuration' pages?

**Comment by stancl** [comment]
https://tenancy.samuelstancl.me/docs/v2/middleware-configuration/

https://github.com/stancl/tenancy-docs/blob/master/docs/navigation.php#L11-L27

**Comment by bakerkretzmar** [comment]
@stancl thanks for your patience, this was tricky 😂 

**TL;DR** - in the `isActive` helper on [line 34 of your `config.php`](https://github.com/stancl/tenancy-docs/blob/master/docs/config.php#L34), make the following change:

```diff
- return ends_with(trimPath($page->getPath()), trimPath($path));
+ return ends_with(trimPath($page->getPath()), trimPath($page->version() . '/' . $path));
```

What's happening here is that `isActive` assumes it's checking the _entire_ path to the current page. In the documentation starter template, it is—we specify the complete path for every page in `navigation.php`:

```php
    'Getting Started' => [
        'url' => 'docs/getting-started',
        'children' => [
            'Customizing Your Site' => 'docs/customizing-your-site',
            'Navigation' => 'docs/navigation',
            // ...
```

With this setup, `isActive` is going to check if `docs/middleware-configuration` ends with `docs/configuration`, which it doesn't, so everything works.

This involves a fair bit of repetition in path names, because we have to write out `docs/` over and over. You've quite thoughtfully hoisted this up into top level `v1` and `v2` keys in your navigation, and you add the version back into each page's URL in the `link` helper, which means for every page you only have to specify the last part of the slug:

```php
    'v1' => [
        'Getting Started' => [
            'url' => 'getting-started',
            'children' => [
                'Installation' => 'installation',
                'Storage Drivers' => 'storage-drivers',
                // ...
```

The default `isActive` helper, however, naively still thinks it's getting the complete path, so on your site it checks if `middleware-configuration` ends with `configuration`, which it does, and that's why you get two active menu items.

One solution to this of course would be adding the version back to every path in your `navigation.php`. But, since you've also got a handy `version` helper set up, it's quicker to just add the version back to the path dynamically right in the `isActive` helper.

**Comment by stancl** [comment]
Ah, thanks a lot for the detailed answer @bakerkretzmar :)
